apps/judicial-system/backend/src/app/modules/defendant/defendant.service.ts

```diff
@@ -305,7 +305,17 @@ export class DefendantService {
         return { delivered: true }
       })
       .catch((reason) => {
-        this.logger.error('Failed to update case with defendant', { reason })
+        const sanitizedReason = JSON.stringify(reason)
+          .replace(
+            /Participant with id: \d{10}/g,
+            'Participant with id: **********',
+          )
+          .replace(/\) gegn(.*?)'/g, ') gegn **********')
+
+        this.logger.error(
+          'Failed to update case with defendant',
+          JSON.parse(sanitizedReason),
+        )
```

Comment:
The error message has already been sanitized in the court service, so there is not need to do it here also.